### DISCUSSION TITLE
Email with attachments not being send

### DISCUSSION THREAD

**Discussion by steve-ref** [emailjs-com/help#77] [Q&A]
Hello, Im new using EmailJS ( paid plan ) with MailJet as Service.
When sending my form, all datas are sent but, no email in my box.
If i use my form without attachment, the mail is sent fine?
Can u help me ?

**Comment by xr0master** [comment]
As a paying customer, you can contact technical support through the EmailJS dashboard.

If you see records in the Email History with the green status, this means that the service successfully processed your request and transferred it to your provider.
Next, I recommend you also visit the provider’s dashboard (MailJet) and check the status of your request.

Don’t forget to check your spam folder since everything may be fine, but additional work is required on the email content or DNS settings.

**Reply by steve-ref** [reply]
If my form is sending a file, MailJet is not receiving my request.
So I guess the problem is between EmailJS and MailJet ?
Thanks for your help btw

**Reply by xr0master** [reply]
I highly recommend you check the Email History records and Events records. It helps a lot to find the issue.

**Reply by steve-ref** [reply]
Yeah I checked them, but all good inside EmailJS so I dont know.

on the other side, In MailJet, I have nothing
